src/model/event.rs

```diff
@@ -268,11 +232,7 @@ pub struct ChannelRecipientRemoveEvent {
 impl CacheUpdate for ChannelRecipientRemoveEvent {
```

Comment:
These `ChannelRecipient*Event` event structures should also be removed. Without groups, they're pointless for bots to listen to the events, or of any significance to the library.